### PR TITLE
feat: completed the image component

### PR DESCRIPTION
**Pull request by therajatg**
please review

### CODE REVIEW

**Review by iamsaurav28 — APPROVED** [review]
looks good to merge

File: styles/component-styles/image.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    display: flex;
+    justify-content: center;
+    border-radius: 2rem;
+    background-color: var(--secondary-container-color);
+    flex-wrap: wrap; 
+    padding: 2rem;
+}
+
+.img-square{
+    object-fit: cover;
```
Comment (vivekmehto) [inline styles/component-styles/image.css:11]: we can give object-fit cover in the reset CSS file.

File: styles/component-styles/image.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+.img-round{
+    object-fit: cover;
+    height: 20rem;
+    width: 20rem;
+    border-radius: 999px;
+}
+
+.img-responsive{
+    object-fit: cover;
+    width: 50%;
```
Comment (vivekmehto) [inline styles/component-styles/image.css:25]: we can use width 100% so that it will totally depend on the divs width, not on the image.

**Review by vivekmehto — COMMENTED** [review]
few suggestions apart from that everything is good

File: styles/component-styles/image.css (hunk 1 above)
Comment (therajatg) [reply styles/component-styles/image.css:11]: I don't have reset.css file.

File: styles/component-styles/image.css (hunk 2 above)
Comment (therajatg) [reply styles/component-styles/image.css:25]: still depends on the div width. Whatever be the div width, image width will be 50% of that.

**Review by vivekmehto — APPROVED** [review]

**Pull request merged**